app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,189 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskServiceTest {
+
+    private static final String TASK_TITLE = "test";
+    private static final String UPDATE_POSTFIX = "!!!";
+
+    private TaskService taskService;
+
+    @BeforeEach
+    void setUp() {
+        // subject
+        taskService = new TaskService();
+
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks{
+        @Nested
+        @DisplayName("Task가 등록되어 있다면")
+        class Context_have_task{
+
+            // 설멸문의 Task가 등록되어 있다면과 똑같은 조건이 되도록 여기에 준비한다.
+            @BeforeEach
+            void prepareTask(){
+                Task task = new Task();
+                task.setTitle(TASK_TITLE);
+
+                taskService.createTask(task);
+            }
+
+            @Test
+            @DisplayName("등록된 모든 Task를 리턴한다")
+            void It_return_tasks(){
+                List<Task> tasks = taskService.getTasks();
+
+                assertThat(tasks).hasSize(1);
+
+                Task task = tasks.get(0);
+                assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
+            }
+        }
+        @Nested
+        @DisplayName("Task가 등록되어 있지 않다면")
+        class Context_have_not_task{
+
+            @BeforeEach
+            void prepareTask(){
+                List<Task> tasks = taskService.getTasks();
+                tasks.forEach(task -> taskService.deleteTask(task.getId()));
+            }
+            @Test
+            @DisplayName("비어 있는 리스트를 리턴한다")
+            void it_return_tasks(){
+                assertThat(taskService.getTasks()).isEmpty();
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("getTask 메소드는")
+    class Describe_getTask{
+        @Nested
+        @DisplayName("등록된 Task의 id가 주어지면")
+        class Context_withValid_id{
+
+            @BeforeEach
+            void prepareTask(){
+                Task task = new Task();
+                task.setTitle(TASK_TITLE);
+
+                taskService.createTask(task);
+            }
+            @Test
+            @DisplayName("해당 Task를 리턴한다")
+            void It_return_task(){
+                Task found = taskService.getTask(1L);
+                assertThat(found.getTitle()).isEqualTo(TASK_TITLE);
+            }
+        }
+
+        @Nested
+        @DisplayName("Task를 찾을 수 없는 id(100L)가 주어지면")
```

Comment:
`100L`이 여기에 들어오면 안됩니다.
```suggestion
        @DisplayName("Task를 찾을 수 없는 id가 주어지면")
```